RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.21.0
+
+- improved: `git machete github anno-prs` adds the URL of the PR
```

Comment:
Hmmm in general... I'd rather still make this behavior controlled by a flag (`--url`) and/or via config key (`machete.github.annotateWithUrl`) so as not to break the behavior for the existing users 😅